Backpack-SwiftUI/Calendar/Classes/Core/CalendarTypeContainerFactory.swift

```diff
@@ -105,4 +107,15 @@ struct CalendarTypeContainerFactory<MonthHeader: View, DayAccessoryView: View>:
         )
 
     }
+    
+    private var adjustedValidRange: ClosedRange<Date> {
```

Comment:
Reviewer note: We need this to display all highlighted dates even if it was in previous month outside of an original valid range. However, it's used only for determining which months we need to show in the calendar.